docker/install-deps

```diff
@@ -43,8 +43,9 @@ if [ -f /usr/bin/yum ]; then
 		gpgkey=http://linuxsoft.cern.ch/cern/slc6X/\$basearch/RPM-GPG-KEY-cern
 		gpgcheck=1
 	EOF
-        PKGLIST=${PKGLIST/ gcc-c++/ devtoolset-2-gcc-c++ devtoolset-2-binutils devtoolset-6-gcc-c++ devtoolset-6-binutils}
+        PKGLIST=${PKGLIST/ gcc-c++/ devtoolset-2-gcc-c++ devtoolset-6-gcc-c++}
```

Comment:
devtoolset-2-gcc-c++ and devtoolset-6-gcc-c++ in the same time?

Reply:
yes, we use the same docker images for all branches (which are prepared by `prepare-docker` command).
so it means that image should contain requirements for all PS versions.
So devtoolset-2 is needed for PS 5.7, and devtoolset-6 for PS 8.0